gno.land/pkg/integration/testing_integration.go

```diff
@@ -326,6 +331,66 @@ func setupGnolandTestScript(t *testing.T, txtarDir string) testscript.Params {
 	}
 }
 
+// `unquote` processes a string slice as one block string. handling quoted
+// phrases and escape sequences.
+func unquote(args []string) ([]string, error) {
+	const quote = '"'
+
+	parts := []string{}
+	inQuote := false
+
+	var part strings.Builder
+	for _, arg := range args {
+		escaped := false
+		for _, c := range arg {
+			// If this character should be escaped, unquote it
+			if escaped {
+				uc, err := strconv.Unquote(`"\` + string(c) + `"`)
```

Comment:
Are you using `Unquote` in order to make sure this is a valid escape character? That's interesting if that's what you're doing; I would have never thought of it.